src/main/java/it/eng/idsa/businesslogic/audit/TrueConnectorEventType.java

```diff
@@ -6,6 +6,8 @@ public enum TrueConnectorEventType {
 	HTTP_REQUEST_RECEIVED("Http request received"),
 	USER_AUTHORIZATION_FAILURE("Authorization failure"),
 	USER_AUTHORIZATION_SUCCESS("Authorization success"),
+	USER_AUTHENTICATION_FAILURE("Authentication failure"),
```

Comment:
Updated also some md file with listed events?
This md might be in TC repo, so we should also check there

Reply:
Honestly, I wasn't aware that md exists, it will be updated in AUDIT.md, and I'll also check in the TC repo.